runtime/lua/vim/fs.lua

```diff
@@ -609,4 +611,56 @@ function M.normalize(path, opts)
   return path
 end
 
+--- @param path string Path to remove
+--- @param ty string type of path
+--- @param recurse? boolean
```

Comment:
```suggestion
--- @param recursive? boolean
```

Suggestion; I feel like that's the more accessible term here.

(`rm` itself uses neither:
> Attempt to remove the file hierarchy rooted in each file argument.

which is just... weird.)

Reply:
Coreutils `rm` has

```
       -r, -R, --recursive
              remove directories and their contents recursively
```

Reply:
fruitrm gotta be different :)